test/test-ghe-backup.sh

```diff
@@ -520,9 +519,24 @@ begin_test "ghe-backup takes backup of Actions settings"
     "actions-launch-app-app-private-key"
   )
 
+  # Add the one options file we included tests for
```

Comment:
```suggestion
  # Add the one optional file we included tests for
```

Reply:
I applied this, but I fat-finged the "close" button so I had to do it manually 😄